### ISSUE TITLE
Move livesim URLs to livesim2

### ISSUE THREAD

**Issue by tobbee** [shaka-project/shaka-player#5534]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**Is your feature request related to a problem? Please describe.**

URLs to the online DASH-IF livesim service at https://livesim.dashif.org is used in a number of places in the code.
I think this is both for testing and for demo. That service will be discontinued before the end of September.
A new service https://livesim2.dashif.org is being developed.

**Describe the solution you'd like**

Replace all URLS in the following way:
* https://livesim.dashif.org/livesim/X with https://livesim2.dashif.org/livesim2/X for live assets
* https://livesim.dashif.org/dash/vod/X with https://livesim2.dashif.org/dash/X for VoD assets

See https://github.com/Dash-Industry-Forum/livesim2/issues/55 for more information.

*Note* There is a limit on the number of requests per IP address per day in the new service. If the build system for Shaka-packager needs more, it may be possible to get a token for it. This feature is not yet developed, but mentioned in 
https://github.com/Dash-Industry-Forum/livesim2/issues/48

**Describe alternatives you've considered**

We will probably redirect the DNS entry to the new instance, and the paths that are currently listed in
the URLs will then work via a redirect, but that is less efficient.

**Additional context**

* A corresponding work is done for dash.js: https://github.com/Dash-Industry-Forum/dash.js/pull/4267
* Please comment on Github on the work on livesim2 if there is any special feature that is important for shaka-player.

**Comment by avelad** [comment]
@tobbee I am reviewing and I see that the limit is 1000 requests per IP, is that correct? I see this very very low to be able to develop/investigate issues, and even more so when the old one livesim, it did not have this limitation.

**Comment by tobbee** [comment]
@avelad Thanks for coming back. I want the service online service to be useful, but DASH-IF had costs of up to $2000/month for traffic from the old livesim, so we need to get that down. A lot of a traffic came from rather few users who run streams 24h/day for extended periods. Hopefully such users will set up their own server since that is now much simpler.

The limit is configurable. Yeah,1000 is low, so I reconfigured it to 10000 (see https://livesim2.dashif.org/config).

On the old service, I introduced a session time limit by making an initial redirect to a URL with a sessionID and start
Time and stop the session after 10min or similar. There was nothing preventing a client to restart after that though.

Anyway, we are considering ways of letting some project have more access by either having a white-list of IP addresses or some sort of tokens. If you have any input/preferences to a solution, it would be good to know. The PR https://github.com/Dash-Industry-Forum/livesim2/issues/48 may be a good place.
